docs/features/externaltools/h5p.md

```diff
@@ -2,45 +2,65 @@
-![Problem 1](../img/externaltools/h5p/problem_1.png)  ![Problem 2](../img/externaltools/h5p/problem_2.png)
-2. Creating the quizzes might either be easy or a lot of work in some cases. Some quizes like the memory are pretty easy and intuitive. Other quizzes like the drag and drop are not intuitive and have some missing translations. This makes it more like a try and error testing instead on creating a task. An example is the problem when you press on the red text. You might think that this just discard the changes or something similar. In practice it deletes the object. 
+There are a couple of things you should be aware of when using this tool. 
+1. The scaling and alignment of items is either automatic and suboptimal or it requires quite some work to do it right. In some cases questions are cut of automatically or answers might overlap each other. Furthermore the scaling is not perfect and the text might get very small or even unreadable in standard settings on small screens. Some examples look like this:
+
+![Problem 1](../img/externaltools/h5p/problem_1.png) 
+![Problem 2](../img/externaltools/h5p/problem_2.png)
+
+2. Creating the quizzes might be a lot of work in some cases. Some quizes like the memory are pretty easy and intuitive. Other quizzes such as the drag and drop are not intuitive and have some missing translations. In these cases one needs a lot of try and error to create a quizz as desired. 
+  Be especially aware of this non-translated red button. You might think that this just discards the changes, but in practice it deletes the entire object. 
 ![Problem 3](../img/externaltools/h5p/problem_3.png)
-A further problem is that the flexibility for the drag and drop quiz is unexpected. You can set the drop zones for each text and you have to do that in order to specify in each drop zone which text(s) should be the correct one. It is not to difficult but it takes more time to get used to compared to the other quizzes.
-3. In some quizzes there are small problem with attending the quiz. In the find the words quiz it is very difficult to select a word in the top row because the mouse it not aligned with the selection on the word field. A second problem we found is that it is also difficult to place larger texts in small drop zones in the drag and drop quiz. In both cases it is not impossible to successfully complete the task but it might be very annoying for the users.
+  A further problem is that the flexibility for the drag and drop quiz is unexpectedly good, but it needs a lot of settings in order to configure it. You can set the drop zones for each text and then specify in each drop zone which text(s) should be the correct one(s). 
+  
+3. In some quizzes there are small problems with attending the quiz. 
+- In the find the words quiz it is very difficult to select a word in the top row because the mouse is not aligned with the selection on the word field. 
```

Comment:
```suggestion
- In the find-the-words quiz it is very difficult to select a word in the top row because the mouse is not aligned with the selection on the word field. 
```